internal/infrastructure/db/sqlite/migration/20250515092010_add_swap_table.up.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE IF NOT EXISTS swaps (
+    id          TEXT    PRIMARY KEY,
+    amount      INTEGER NOT NULL,
+    date        TEXT    NOT NULL,
+    "to"        TEXT    NOT NULL,
+    "from"      TEXT    NOT NULL,
```

Comment:
Let's store the timestamp as int64. Storing it as a string date must be done taking into account the timezone, it just adds complexity and leads to the current issue: when swaps are sourced from the db, their timestamp can be +/- N hours depending on the timezone I am currently in.  Let's also give names to the currency fields that don't go in conflict with reserved words.

```suggestion
    timestamp        INTEGER    NOT NULL,
    from_currency        TEXT    NOT NULL,
    to_currency      TEXT    NOT NULL,
```